src/Eris/Generator/SuchThatGenerator.php

```diff
@@ -4,11 +4,17 @@
 use Eris\Generator;
 use LogicException;
 
+/**
```

Comment:
Boy scout <img src="http://www.gadlerner.it/wp-content/uploads/2014/06/renzi-premier.jpg" />